PrebidMobile/AdUnits/Parameters/VideoParameters.swift

```diff
@@ -54,8 +54,12 @@ public class VideoParameters: NSObject {
     public var startDelay: Signals.StartDelay?
     
     /// Placement type for the impression.
+    @available(*, deprecated, message: "This property is deprecated. Please, use plcmnt instead.")
```

Comment:
We shouldn't deprecate this property in the SDK. 

The reason is we still should support OpenRTB 2.5. 

The publishers will have to set both properties `placement` and  `plcmnt`.